### ISSUE TITLE
Allow Queen Bee to be the first move.

### ISSUE THREAD

**Issue by guillermo-morini-ef** [jonthysell/Mzinga#93]
As a player
When I go to place the first piece
Then I should be able to place the queen as my first move.

**Comment by JHonaker** [comment]
While this is not an official rule in the game box, the community has decided to disallow Queen placements first. They found that it led to an inordinate amount of draws. It's referred to as Tournament Rules online. I think this only implements TR.

**Comment by jonthysell** [comment]
Yes, I implemented the tournament rules (no queen-bee first moves) because even the game's creator endorsed the rule change.

Because of how I architected the code, I just never got around to figuring out rules variants.

**Comment by vovanz** [comment]
@jonthysell do you thin it is possible to add an option to [GameTypeString](https://github.com/jonthysell/Mzinga/wiki/UniversalHiveProtocol#gametypestring) in UHP to choose the opening type?

Something like `Base+StandardOpening` or `Base+TournamentOpening`.
